modules/theme/src/themes/default/elements/header.overrides

```diff
@@ -29,6 +29,10 @@ h6.ui.header .sub.header {
   font-size: @h6SubHeaderFontSize;
 }
 
+h5.ui.header {
```

Comment:
Other header variations should be aligned with this font-weight.

Reply:
If I'm correct this is only applied to `h5` right? header variations can depend on `font-size` but for `font-weight` I don't think it's correct. Or am I missing something here?

Reply:
We have to increase the font-weight relatively for the other variations as well. You can fire up the react-components and check how the different font variations look. With only `h5` having a increased `font-weight`, it'll look weird IMO.

Reply:
You can change this variable to change the font-weight for all the headers.